src/redux/actions/opponents.ts

```diff
@@ -7,7 +7,7 @@ export enum OpponentActionType {
 };
 
 export const OpponentAction = {
-  syncOpponents: (opponents: string[]) => ({
+  syncOpponents: (opponents: any[]) => ({
```

Comment:
Why not
`import { Opponent } from '$ROOT_DIR/src/reducers/opponents';`
and then 
`syncOpponents: (opponents: Opponent[]) => ({`
?